frontend/src/pages/journal/components/journal-sidebar/journal-sidebar.tsx

```diff
@@ -49,6 +72,11 @@ const JournalSidebar: React.FC = () => {
                 onClick={handleSelectJournalEntry(journalEntry.id)}
                 isActive={selectedJournalEntry?.id === journalEntry.id}
                 key={journalEntry.id}
+                linkTo={
+                  `${AppRoute.JOURNAL}/${journalEntry.id}` as ValueOf<
```

Comment:
Create full path and with replace put the value here. The example on Chant functionality's PR